### PR TITLE
Captive Portal - Deprecate jQuery

### LINKED ISSUE #7131: jquery-1.11.3.min.js is vulnerable to XSS
**Describe the bug**
`html/common/jquery-1.11.3.min.js` is loaded by `html/captive-portal/templates/layout.html` when you go on captive portal and functions are used by JS files used on captive portal.

This library is vulnerable to XSS. We need to upgrade to 3.5.0 at least.

### PR DESCRIPTION
**Pull request by satkunas**
# Description
 * remove jQuery (and prototype) from `html/captive-portal`, `html/common`
 * replace qrcode dependency w/ no sub-dependancies

# Impacts
 * May impact older browser compatibility.
 * Captive Portal workflows not tested during removal.

# Code / PR Dependencies
none

# Delete branch after merge
YES 

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

## Bug Fixes
* Fixes #7131 

## Enhancements
* Removed jQuery dependency in captive portal.

### CODE REVIEW

**Comment by satkunas** [comment]
Browser support w/ replacement fn's

Screen Shot 2022-09-16 at 4 36 36 PM

File: html/captive-portal/content/billing/authorizenet.js [hunk 1]
```diff
@@ -31,7 +30,7 @@
               for (var i = 0; i < response.messages.message.length; i++) {
                   console.log(response.messages.message[i].code + ": " + response.messages.message[i].text);
               }
-              var $form = $('#payment-form');
+              var $form = document.getElementById('payment-form');
               $form.find('.payment-errors p').text('Unable to proceed with payment please contact your service provider');
```
Comment (julsemaan) [inline html/captive-portal/content/billing/authorizenet.js]: A very newb question. Back when I did JS, you couldn't use very advanced functions on its result (like find/text/etc). Just double checking that the find method (and text) are indeed defined for the object returned by getElementById now

**Comment by julsemaan** [comment]
> Browser support w/ replacement fn's
> 
> Screen Shot 2022-09-16 at 4 36 36 PM

Did you implement a check in the portal where if a device doesn't meet the new requirements, it will show a very clear warning at the top of the portal page?

File: html/captive-portal/content/billing/authorizenet.js (hunk 1 above)
Comment (satkunas) [reply html/captive-portal/content/billing/authorizenet.js]: `.find()` in this context is a jQuery function that was missed. I'll revisit.

**Comment by satkunas** [comment]
> > Browser support w/ replacement fn's
> > Screen Shot 2022-09-16 at 4 36 36 PM
> 
> Did you implement a check in the portal where if a device doesn't meet the new requirements, it will show a very clear warning at the top of the portal page?

There currently appears to be a mechanism to display such a message. I did not review its function.

**Comment by julsemaan** [comment]
> > > Browser support w/ replacement fn's
> > > Screen Shot 2022-09-16 at 4 36 36 PM
> > 
> > 
> > Did you implement a check in the portal where if a device doesn't meet the new requirements, it will show a very clear warning at the top of the portal page?
> 
> There currently appears to be a mechanism to display such a message. I did not review its function.

This needs to be added to this PR as its a hard requirement for getting this in the hands of customers so that JS issues don't go silently and the portal is just broken without any feedback

**Pull request merged**